packages/mdc-list/README.md

```diff
@@ -1,49 +1,23 @@
-<!--docs:
-title: "Lists"
-layout: detail
-section: components
-excerpt: "Lists present multiple line items vertically as a single continuous element."
-iconId: list
-path: /catalog/lists/
--->
-
 # Lists
 
-<!--<div class="article__asset">
```

Comment:
Same with these comments (see the [template](https://github.com/material-components/material-components-web/blob/master/docs/code/readme_template.md))